framework/CMakeLists.txt

```diff
@@ -1,21 +1,32 @@
 include("${AREG_BASE}/areg/CMakeLists.txt")
 include("${AREG_BASE}/mcrouter/CMakeLists.txt")
 
+# Adding  source files to a single executable.
+set(EXE mcrouter${AREG_BIN_EXT})
+add_executable(${EXE} ${mcrouter_SRC})
 
-if(AREG_BINARY MATCHES static)
-    add_library(areg STATIC ${areg_SRC} ${mcrouter_SRC})
```

Comment:
This is an issue, and it is good that it is being fixed now

Reply:
I meant building mcrouter source files to areg library will cause issues

Reply:
Yes, sorry that was my mistake :)